cmd/uhc/common/args.go

```diff
@@ -0,0 +1,8 @@
+package common
```

Comment:
Add the copyright header.

I'd prefer to have this in a package inside the `pkg` directory. Could be `pkg/options`, for example. The reason is that I'd prefer to stick to the rule that every package insde the `cmd` directory corresponds exactly to a command.